bitnami/airflow/templates/configmap-dags.yaml

```
@@ -4,10 +4,17 @@ kind: ConfigMap
 metadata:
   name: {{ template "airflow.fullname" . }}-dag-files
   labels:
+    {{- if .Values.commonLabels }}
+    {{- include "airflow.tplValue" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
     app.kubernetes.io/name: {{ include "airflow.name" . }}
     helm.sh/chart: {{ include "airflow.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+  {{- if .Values.commonAnnotations }}
+  annotations:
+    {{- include "airflow.tplValue" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
```

```suggestion
    annotations: {{- include "airflow.tplValue" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
```
This could be only one line as you are already using `nindent`. Sorry if it seems like nick-picking but it is just to maintain standarization across our charts.